_data/authors.yml

```diff
@@ -309,3 +309,10 @@ annettebieniusa:
   full_name: "Annette Bieniusa"
   portrait: "/img/media/speakers/annettebieniusa.jpg"
   bio: "Annette is a lecturer and senior researcher at the Technische Universität Kaiserslautern. Her research interests include semantics of concurrent and distributed programming, with a focus on replication, synchronization, and how they are reflected on programming language level. Annette was involved in several national and international research projects, most recently the in the EU-Projects “SyncFree: Large-scale Computation without Synchronization” and “Lightkone: Lightweight computation for networks at the edge“."
+gvolpe:
+  full_name: "Gabriel Volpe"
+  portrait: "/img/media/speakers/gvolpe.jpg"
+  github: "gvolpe"
+  twitter: "volpegabriel87"
+  bio: "FP developer Scala | Haskell"
```

Comment:
Bio is not necessary, it will not be rendered below the blog post.

Reply:
Okay, will remove. Copy / pasted from others :smile: